smart_card_connector_app/src/window-main.js

```diff
@@ -45,19 +45,21 @@ const logger = GSC.Logging.getScopedLogger('ConnectorApp.MainWindow');
 
 goog.log.info(logger, 'The main window is created');
 
-goog.events.listen(
-    goog.dom.getElement('close-window'), goog.events.EventType.CLICK,
-    closeWindowClickListener);
+//revisitcode
+// goog.events.listen(
+//     goog.dom.getElement('close-window'), goog.events.EventType.CLICK,
+//     closeWindowClickListener);
 
+//revisitcode
 GSC.ConnectorApp.Window.AboutShowing.initialize();
-GSC.ConnectorApp.Window.AppsDisplaying.initialize();
-GSC.ConnectorApp.Window.DevicesDisplaying.initialize();
+// GSC.ConnectorApp.Window.AppsDisplaying.initialize();
+// GSC.ConnectorApp.Window.DevicesDisplaying.initialize();
 GSC.ConnectorApp.Window.HelpShowing.initialize();
 GSC.ConnectorApp.Window.LogsExporting.initialize();
 
 GSC.I18n.adjustAllElementsTranslation();
 
-GSC.PopupWindow.Client.showWindow();
+// GSC.PopupWindow.Client.showWindow();
```

Comment:
BTW, now that we don't explicitly make the window visible and it's shown from the very beginning, does this introduce visible flickering? I.e., are you able to see the window being initially opened empty, and in the next moment populated with data?

If that's happening indeed, that'd be unfortunate. I don't have an immediate proposal about how to fix it though.

Reply:
Nope, no flickering. I believe this function was for the app, but as an extension the window is opened right away when clicking on the icon